internal/proxy/task.go

```diff
@@ -1474,10 +1482,7 @@ func (st *searchTask) PreExecute(ctx context.Context) error {
 
 	st.Base.MsgType = commonpb.MsgType_Search
 
-	schema, err := globalMetaCache.GetCollectionSchema(ctx, collectionName)
-	if err != nil { // err is not nil if collection not exists
-		return err
-	}
+	schema, _ := globalMetaCache.GetCollectionSchema(ctx, collectionName)
```

Comment:
why ignore this err ?

Reply:
we have checked if collection exists in system before (using globalMetaCache.GetCollectionID)